src/engraving/rendering/dev/slurtielayout.cpp

```diff
@@ -2495,8 +2495,8 @@ void SlurTieLayout::computeMidThickness(SlurTieSegment* slurTieSeg, double slurT
     const double mag = slurTieSeg->staff() ? slurTieSeg->staff()->staffMag(slurTieSeg->slurTie()->tick()) : 1.0;
     const double minTieLength = mag * slurTieSeg->style().styleS(Sid::MinTieLength).val();
     const double shortTieLimit = mag * 4.0;
-    const double minTieThickness = mag * (0.15 * slurTieSeg->spatium() - slurTieSeg->style().styleMM(Sid::SlurEndWidth));
-    const double normalThickness = mag * (slurTieSeg->style().styleMM(Sid::SlurMidWidth) - slurTieSeg->style().styleMM(Sid::SlurEndWidth));
+    const double minTieThickness = mag * (0.15 * slurTieSeg->spatium() - slurTieSeg->style().styleMM(Sid::TieEndWidth));
+    const double normalThickness = mag * (slurTieSeg->style().styleMM(Sid::TieMidWidth) - slurTieSeg->style().styleMM(Sid::TieEndWidth));
```

Comment:
The only remark I might have about this is that it is not really guaranteed that this method won't be called for slur segments in the future, which would not go well with the tie-specific Sids...
If you change the signature of the method like this:
```diff c++
-void SlurTieLayout::computeMidThickness(SlurTieSegment* slurTieSeg, double slurTieLengthInSp)
+void SlurTieLayout::computeMidThickness(TieSegment* tieSeg, double slurTieLengthInSp)
```
then I think it should still compile, while it _is_ made clear that this method is currently for ties only. 

(When someone wants to use it for slurs in the future, they will need to change it back to accept slur segments too, but then that someone will surely not forget to look into the body of the method and then see the tie-specific Sid being used.)